app/services/campaign_page_builder.rb

```diff
@@ -26,9 +26,15 @@ def page
   end
 
   def create_plugins
-    page.liquid_layout.partial_refs.each do |partial, ref|
-      plugin_name = LiquidPartial.find_by(title: partial).plugin_name
-      Plugins.create_for_page(plugin_name, page, ref)
+    if default
+      Plugins.registered.each do |plugin|
+        Plugins.basic_create_for_page(plugin, page)
+      end
```

Comment:
I was having difficulty experimenting with new partials with this system. If you don't mind, I'd like to keep this in for a while longer (for development), at least until you've got template switching working. 

Simply set the env to false in your local environment to maintain the standard ('ref') behaviour.